### PR TITLE
Add incubator calls participation guidelines

### PR DESCRIPTION
**Pull request by syg**
Thoughts welcome.

### CODE REVIEW

File: incubator-calls.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+# TC39 Incubator Call Participation Guidelines
+
+A model of the TC39 working mode is this flowchart:
+
+![TC39 Working Mode](images/tc39-working-mode.svg)
+
+Incubator calls seek to speed up the process by having a sanctioned bi-weekly
```
Comment (codehag) [inline incubator-calls.md:7]: bi-weekly can be confusing. maybe "every two weeks"?

File: incubator-calls.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+In alignment with the guiding principle, core stakeholders should prioritize
+giving feedback that they feel would slow down or endanger eventual stage
+advancement. That said, the usual guidelines about feedback apply: the feedback
+should be actionable and there should be novel information available.
+
+Core stakeholders should challenge out-of-scope limitations if they
+disagree. This can be done out-of-band or on the GitHub thread.
+
+Example feedback to add to the agenda ahead of the meeting:
+- Feedback (from <NAME>): FinalizationRegistry should allow users to express backpressure.
```
Comment (codehag) [inline incubator-calls.md:72]: great idea!

File: incubator-calls.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+there are no hard timeboxes. Instead, the facilitators and champions may ask to
+move on to the next topic at their discretion. Discussions are facilitated by
+TCQ, like in plenary. Notes are taken, like in plenary.
+
+Meeting facilitators are specifically empowered to keep the discussion within
+the scope of the current agenda item. For example, if we're talking about one
+area of concern, and participants bring up another, facilitators may decide to
+queue that new area to be discussed later in the meeting. Or, if participants
+bring up an explicitly out-of-scope item, the facilitators may remind
+participants about the meeting scope.
```
Comment (codehag) [inline incubator-calls.md:126]: If something is out-of-scope and not previously brought up in discussion, we can have an action item to open an issue or open a discussion on discourse (for example the discussion around option bags)

File: incubator-calls.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+Remember, the guiding principle is to present and receive relevant feedback in a
+more timely manner. When in doubt, yield time to other participants to present
+their feedback instead of attempting to resolve the feedback in the
+moment. Resolving the issue should ideally be done asynchronously, or in plenary
+itself, if particularly controversial.
+
+# After the Call
+
+Participants are encouraged to review the notes, which will be published 7 days
```
Comment (codehag) [inline incubator-calls.md:147]: Where will this be published?

Comment (syg) [reply incubator-calls.md:147]: I was hoping a new top-level folder in tc39/notes called "incubator-calls".

File: incubator-calls.md (hunk 3 above)
Comment (syg) [reply incubator-calls.md:126]: +1

File: incubator-calls.md (hunk 1 above)
Comment (syg) [reply incubator-calls.md:7]: +1

**Review by invisiblepancake — COMMENTED** [review]
Added New File: images/tc39-working-mode.svg

An SVG file illustrating the TC39 working mode was added.
Added New File: incubator-calls.md

Created new markdown file outlining the TC39 Incubator Call Participation Guidelines.
Changes include a detailed guide on setting the agenda, participant categories, and the process before, during, and after the calls.
Key Changes in incubator-calls.md
Introductory Section:
A model flowchart of the TC39 working mode.
General guiding principles of the incubator calls for presenting and receiving relevant feedback for stage advancement.
Participant Categories:
Definitions for Champions, Core stakeholders, Contributors, and Facilitators.
Agenda Setting Before the Call:
Detailed instructions for Champions, Core stakeholders, and Contributors on setting and reviewing the agenda.
During the Call:
Process for facilitating discussions, managing agenda items, and identifying action items.
After the Call:
Encouragement for participants to review published notes.
Review Suggestions
Clarity and Consistency:
Ensure the language is clear and consistent throughout the document.
Links and References:
Verify that all links, such as GitHub issues and proposals, are correct and functional.
Formatting:
Check markdown formatting for proper display, especially for bullet points and headings.
Content Accuracy:
Confirm that the guidelines accurately reflect the intended process and roles within the TC39 incubator calls.
If you have any specific questions or need further assistance with the review, feel free to ask!

**Pull request merged**